Sources/AccessibilitySnapshot/Core/Swift/Assets/ru.lproj/Localizable.strings

```diff
@@ -0,0 +1,80 @@
+
+/* Format for hint describing how to use elements with the 'adjustable' accessibility trait; param0: the existing hint */
+"trait.adjustable.hint_format" = "%@. Смахните вверх или вниз одним пальцем, чтобы настроить значение.";
+
+/* Description for the 'image' accessibility trait */
+"trait.image.description" = "Изображение.";
+
+/* Description for the 'search field' accessibility trait */
+"trait.search_field.description" = "Поле поиска.";
+
+/* Description for the 'switch button' accessibility trait, when the switch is on */
+"trait.switch_button.state_on.description" = "Кнопка-переключатель. Вкл.";
+
+/* Description for the 'switch button' accessibility trait, when the switch is off */
+"trait.switch_button.state_off.description" = "Кнопка-переключатель. Выкл.";
+
+/* Description for the 'switch button' accessibility trait, when the state of the switch cannot be determined */
+"trait.switch_button.state_unspecified.description" = "Кнопка-переключатель.";
+
+/* Hint describing how to use elements with the 'switch button' accessibility trait */
+"trait.switch_button.hint" = "Коснитесь дважды, чтобы переключить настройку.";
+
+/* Format for hint describing how to use elements with the 'switch button' accessibility trait; param0: the existing hint */
+"trait.switch_button.hint_format" = "%@. Коснитесь дважды, чтобы переключить настройку.";
+
+/* Format for the description of an element in a series; param0: the description of the element, param1: the index of the element in the series, param2: the number of elements in the series */
+"context.series.description_format" = "%1$@ %2$@ из %3$@.";
+
+/* Format for the description of the height of a cell in a table; param0: the number of rows the cell spans */
+"context.data_table.row_span_format" = "Охватывает столбцов: %@.";
+
+/* Format for the description of the width of a cell in a table; param0: the number of columns the cell spans */
+"context.data_table.column_span_format" = "Охватывает строк: %@.";
+
+/* Format for the description of the vertical location of a cell in a table; param0: the row in which the cell resides */
+"context.data_table.row_format" = "Ряд %@.";
+
+/* Format for the description of the horizontal location of a cell in a table; param0: the column in which the cell resides */
+"context.data_table.column_format" = "Столбец %@.";
+
+/* Description of the first element in a list */
+"context.list_start.description" = "Начало списка.";
+
+/* Description of the last element in a list */
+"context.list_end.description" = "Конец списка.";
+
+/* Description of the first element in a landmark container */
+"context.landmark_start.description" = "Ориентир.";
+
+/* Description of the last element in a landmark container */
+"context.landmark_end.description" = "Конец строки.";
+
+/* Description for an accessibility element indicating that it has custom actions available */
+"custom_actions.description" = "Доступны действия";
```

Comment:
```suggestion
"custom_actions.description" = "Доступные действия";
```

Reply:
Here is an adverb. Proof:

![IMG_5011](https://user-images.githubusercontent.com/3120680/171997254-54dfa5ad-576b-4096-88e9-468f7742e1e3.PNG)